testing/src/main/java/org/oppia/testing/TestingUtils.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.testing
+
+import android.view.View
+import android.widget.EditText
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers
+import org.hamcrest.CoreMatchers
+import org.hamcrest.Matcher
+
+/**
+ * Appends the specified text to a view. This is needed because Robolectric doesn't seem to
+ * properly input digits for text views using 'android:digits'. See
+ * https://github.com/robolectric/robolectric/issues/5110 for specifics.
+ */
+
+fun appendText(text: String, testCoroutineDispatchers: TestCoroutineDispatchers): ViewAction {
```

Comment:
How about we create a particular class here and do the constructor injection for `testCoroutineDispatchers`. Also, try to run the test cases with your changes to see if test cases work fine in both the espresso as well as robolectric. 
Take the example of `ProfileTestHelper`. 
This way I think if more such repetitive methods occur in future which is more inclined towards in use of all the test cases, we can put them here.